packages/form-js-viewer/test/spec/custom/index.js

```diff
@@ -2,7 +2,7 @@ import { formFieldClasses } from '../../../src/render/components/Util';
 
 const type = 'button';
 
-function CustomButtonRenderer(props) {
```

Comment:
Just a recommendation: The PR itself is already quite huge; maybe we can leave these unrelated changes out of it?

That does not mean that I don't appreciate the "refactor on the run" approach 🙂

Reply:
I fixed this because it was breaking our tests, due to button now not being keyed. Figured I might as well fix it completely :).

Reply:
Fair enough! 👍